dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -24,30 +39,32 @@ typedef struct _StaticShellEntry_
     HKEY hkClass;
 } StaticShellEntry, *PStaticShellEntry;
 
+#define DCM_FCIDM_SHVIEW_OFFSET 0x7000 // Offset from the menu ids in the menu resource to FCIDM_SHVIEW_*
 
 //
 // verbs for InvokeCommandInfo
 //
-struct _StaticInvokeCommandMap_
+static const struct _StaticInvokeCommandMap_
 {
     LPCSTR szStringVerb;
-    UINT IntVerb;
+    WORD IntVerb;
+    SHORT DfmCmd;
```

Comment:
Just use UINT as a type, and you will avoid a ton of casts: the (SHORT) below, and the recasting into (int) elsewhere in the code. (And don't tell me you chose SHORT because it makes the structure apparently smaller...)

Reply:
> And don't tell me you chose SHORT because it makes the structure apparently smaller

I am telling you that!

> you will avoid a ton of casts

There is just one cast but and I might even be able to remove that. The cast is there because the documentation for DFM_GETDEFSTATICID used to be poor, not because of the SHORT.